sklearn/model_selection/tests/test_search.py

```diff
@@ -591,71 +599,190 @@ def test_param_sampler():
         assert_equal([x for x in sampler], [x for x in sampler])
 
 
-def test_randomized_search_grid_scores():
+def test_search_results():
     # Make a dataset with a lot of noise to get various kind of prediction
     # errors across CV folds and parameter settings
     X, y = make_classification(n_samples=200, n_features=100, n_informative=3,
                                random_state=0)
 
-    # XXX: as of today (scipy 0.12) it's not possible to set the random seed
-    # of scipy.stats distributions: the assertions in this test should thus
-    # not depend on the randomization
-    params = dict(C=expon(scale=10),
-                  gamma=expon(scale=0.1))
-    n_cv_iter = 3
+    n_folds = 3
+    n_grid_points = 6
+    params = [dict(kernel=['rbf', ], C=[0.01, 0.1], gamma=[0.1, 1]),
+              dict(kernel=['linear', ], degree=[1, 2])]
+    assert_equal(len(ParameterGrid(params)), n_grid_points)
+    grid_search = GridSearchCV(SVC(), cv=n_folds, iid=False,
+                               param_grid=params)
+    grid_search.fit(X, y)
+    grid_search_iid = GridSearchCV(SVC(), cv=n_folds, iid=True,
+                                   param_grid=params)
+    grid_search_iid.fit(X, y)
```

Comment:
Ah yes, thanks for reminding... I wanted to ask if we can remove [that](https://github.com/scikit-learn/scikit-learn/pull/6697/files#diff-8d972a7f847735e5544e1fec73d71e1cL279)?